app/assets/stylesheets/presentation.scss

```diff
@@ -0,0 +1,3 @@
+.presentation-upload {
+  cursor: pointer;
```

Comment:
This is fine for now but we really should add a class called `pointer` to avoid having to do this